features/manage-position/Create.tsx

```diff
@@ -235,44 +235,47 @@ const Create = () => {
                 error={resultantTokensBelowMin}
                 helperText={
                   resultantTokensBelowMin &&
-                  `You must maintain at least ${minSponsorTokensFromWei} ${tokenSymbol} in your position`
+                  `Below minimum of ${minSponsorTokensFromWei}`
                 }
                 onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
                   setTokens(e.target.value)
                 }
               />
             </Grid>
-            <Grid item xs={4}>
-              <Box py={1}>
+            <Grid item md={4} xs={6}>
+              <Box py={0}>
                 {needAllowance && (
                   <Button
+                    fullWidth
                     variant="contained"
                     onClick={setMaxAllowance}
-                    style={{ marginRight: `12px` }}
                   >
                     Max Approve
                   </Button>
                 )}
-                <Button
-                  variant="contained"
-                  onClick={mintTokens}
-                  disabled={
-                    needAllowance ||
-                    transactionCRBelowGCR ||
-                    balanceBelowCollateralToDeposit ||
-                    resultantCRBelowRequirement ||
-                    resultantTokensBelowMin ||
-                    collateralToDeposit <= 0 ||
-                    tokensToCreate <= 0
-                  }
-                >
-                  {`Create ${tokensToCreate} ${tokenSymbol} with ${collateralToDeposit} ${collSymbol}`}
-                </Button>
+                {!needAllowance && (
+                  <Button
+                    fullWidth
+                    variant="contained"
+                    onClick={mintTokens}
+                    disabled={
```

Comment:
If you're only going to show this button when they don't need allowance, can you remove the `needAllowance` from the disabled={} prop since its redundant now?